src/SwedbankPay/Core/Order.php

```diff
@@ -177,8 +177,8 @@ public function getCardHolderInformation()
             'lastName' => $this->getBillingLastName(),
             'email' => $this->getBillingEmail(),
             'msisdn' => $this->getBillingPhone(),
-            'homePhoneNumber' => $this->getBillingPhone(),
-            'workPhoneNumber' => $this->getBillingPhone(),
+            //'homePhoneNumber' => $this->getBillingPhone(),
+            //'workPhoneNumber' => $this->getBillingPhone(),
```

Comment:
Will just removing these make them optional?

Reply:
I'll hope that we get back their :-) Because they are part of cardholder information.
![Screenshot](https://user-images.githubusercontent.com/6286270/84496232-666ecf80-acce-11ea-8485-57cc66eb0e94.png)

Reply:
So the phone numbers should be set if available, but not be required. I don't see how this PR does anything about that? This PR just removes the phone numbers completely?